bitstreams.md

```diff
@@ -102,6 +102,9 @@ Example: <http://dspace7.4science.it/dspace-spring-rest/api/core/bitstreams/8d33
 
 It returns the actual content (bits) described by the bitstream
 
+Supported Request Parameter:
+* **token**: optional parameter that allows downloads for restricted files, a token needs to be requested from the [following endpoint](authentication.md#Request-download-token) 
```

Comment:
This link looks to be broken now, as it refers to "Request download token" and that's been renamed ;)

Reply:
You are right, fixed it.